config/controller.yaml

```diff
@@ -126,6 +126,9 @@ spec:
           # User 65532 is the distroless nonroot user ID
           runAsUser: 65532
           runAsGroup: 65532
+          fsGroup: 65535
+          seLinuxOptions:
+            role: "RunAsAny"
```

Comment:
Does the controller truly need `RunAsAny`? If not, we would be able to use a restricted PodSecurityAdmission which would be preferable

Reply:
I tested with RunAsUser and it shall work.

Reply:
Great! In that case, can you add a "restricted" pod security admission label to the tekton-pipelines namespace? We also can remove the docs, since there won't be any additional action needed for cluster operators.

Reply:
I was wondering if we should remove the docs. Thinking about the use where there are some customized enforcement by PSP but not covered by our tekton/pipeline version of 101-podsecuritypolicy.yaml. Shall we provide any instructions?

Reply:
Are there any aspects of our old PSP that are not covered by the configuration you've added? I don't think there are, but having a table as I suggested would be really helpful to verify that that's the case.